src/helpers.js

```diff
@@ -11,9 +11,17 @@ export const helpers = {
     else
       return 'btn btn-info'
   },
-  rowHtml(item) {
-    var table_rows = '<tr><td class="details-control"><button class="' + helpers.status_btn(item.status) + '" style="width:120px">' + item.status + '</td>'
-      + '<td>' + item.gid + '</td><td>' + item.rule_title + '</td><td>' + item.severity + '</td><td>' + item.nist +'</td></tr>';
+  rowHtml(control) {
```

Comment:
This seems like it should be a set of render functions or even Functional Components
https://www.youtube.com/watch?v=KS4eizPXRCQ
and the section in the VueMastery Advanced Components course